migrations/2025-04-09-123352_cursor_pagination_indexes/up.sql

```diff
@@ -10,3 +10,41 @@ CREATE INDEX idx_post_actions_like_score ON post_actions (post_id, like_score, p
 WHERE
     like_score IS NOT NULL;
 
+-- Fixing the community sorts for an id tie-breaker
+DROP INDEX idx_community_lower_name;
+
+DROP INDEX idx_community_hot;
+
+DROP INDEX idx_community_published;
+
+DROP INDEX idx_community_subscribers;
+
+DROP INDEX idx_community_title;
+
+DROP INDEX idx_community_users_active_month;
+
+CREATE INDEX idx_community_lower_name ON community USING btree (lower((name)::text), id);
+
+CREATE INDEX idx_community_hot ON community USING btree (hot_rank DESC, id);
+
+CREATE INDEX idx_community_published ON community USING btree (published DESC, id);
+
+CREATE INDEX idx_community_subscribers ON community USING btree (subscribers DESC, id);
+
+CREATE INDEX idx_community_title ON community USING btree (title, id);
+
+CREATE INDEX idx_community_users_active_month ON community USING btree (users_active_month DESC, id);
+
+-- Create a few missing ones
+CREATE INDEX idx_community_users_active_half_year ON community USING btree (users_active_half_year DESC, id);
+
+CREATE INDEX idx_community_users_active_week ON community USING btree (users_active_week DESC, id);
+
+CREATE INDEX idx_community_users_active_day ON community USING btree (users_active_day DESC, id);
+
+CREATE INDEX idx_community_subscribers_local ON community USING btree (subscribers_local DESC, id);
+
+CREATE INDEX idx_community_comments ON community USING btree (comments DESC, id);
+
+CREATE INDEX idx_community_posts ON community USING btree (posts DESC, id);
+
```

Comment:
Make `id` match the other column's direction, even if it's ascending (the default).

Reply:
still need to make both columns have the same direction in these indexes: idx_community_lower_name, idx_community_title

they already have the same direction in queries

Reply:
Alright, I made those ones both `DESC` also.